Sming/Wiring/WVector.h

```diff
@@ -266,55 +265,49 @@ unsigned int Vector<Element>::size() const
 }
 
 template <class Element>
-void Vector<Element>::addElement(const Element &obj)
+bool Vector<Element>::addElement(const Element &obj)
 {
-  if (_size == _capacity)
-    ensureCapacity(_capacity + _increment);
-  if (_size < _capacity)
-    _data[ _size++ ] = new Element(obj);
+  if (!ensureCapacity(_size + 1)) return false;
```

Comment:
Can you please at least put the `return` on the next line with a bit of offset ? Best would be to use curly braces. 
Apply those changes to the other places where you have a return on the same line.

Reply:
It would be simplest to just fix all the coding style issues in this file, i.e. just apply the Sming CS and fix all the style issues. The Arduino Esp8266 doesn't use this class (uses std:;vector instead) so it's really a Sming-only file ATM.